smsdk/client.py

```diff
@@ -549,34 +630,31 @@ def get_line_data(
             limit=limit, offset=offset, **kwargs
         )
 
-    xAxisTime = {
-        "unit": "",
-        "type": "datetime",
-        "data_type": "datetime",
-        "stream_types": [],
-        "raw_data_field": "",
-        "id": "endtime",
-        "title": "Time",
-        "isEnabled": True,
-    }
-    one_week_relative = {
-        "time_type": "relative",
-        "relative_start": 1,
-        "relative_unit": "week",
-        "ctime_tz": "America/Los_Angeles",
-    }
-
     def create_share_link(
         self,
-        assets,
-        chartType,
-        yAxis,
-        xAxis=xAxisTime,
-        model="cycle",
-        time_selection=one_week_relative,
         *args,
         **kwargs,
     ):
+        print(f"DebugInfo:: create_share_link")
+        assets = kwargs.pop("assets", None)
+        if assets is None and args:
+            assets = args[0]
+            args = args[1:]
+
+        chartType = kwargs.pop("chartType", None)
+        if chartType is None and args:
+            chartType = args[0]
+            args = args[1:]
+
+        yAxis = kwargs.pop("yAxis", None)
+        if yAxis is None and args:
+            yAxis = args[0]
+            args = args[1:]
+
```

Comment:
xAxis, model and time_selection are all now unavailable via args

Reply:
Fixed the code.